modules/cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class.adoc

```diff
@@ -27,45 +57,56 @@ spec:
       type: RuntimeDefault
   containers:
   - name: qos-demo-ctr
-    image: <image-pull-spec>
+    image: quay.io/openshifttest/hello-openshift:openshift <1>
     resources:
       limits:
-        memory: "200Mi"
-        cpu: "1"
+        memory: "200Mi" <2>
+        cpu: "1" <3>
       requests:
-        memory: "200Mi"
-        cpu: "1"
+        memory: "200Mi" <4>
+        cpu: "1" <5>
     securityContext:
       allowPrivilegeEscalation: false
       capabilities:
         drop: [ALL]
 ----
+<1> This example uses a public `hello-openshift` image. Replace with the image that you want to use for your workload.
```

Comment:
```suggestion
<1> This example uses a public `hello-openshift` image. Replace that value with the image that you want to use for your workload.
```

Reply:
Thanks. Actually I might just remove the second sentence. Given the example image is a `hello-openshift` image, I think it's pretty obvious you can replace it with whatever image you want.